src/__tests__/category/category.mutation.test.js

```diff
@@ -0,0 +1,201 @@
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const categoryService = require('../../modules/category/category.service')
+const categoryResolver = require('../../modules/category/category.resolver')
+const client = require('../../utils/apolloClient');
+
+require('dotenv').config();
+
+let categoryId
+
+describe('mutations', () => {
+    test('should create new category', async () => {
+        const res = await client.mutate({ 
+            mutation: gql`
+                mutation {       
+                    addCategory(category: {
+                        name: [{
+                            lang: "eng",
+                            value: "Test"
+                        },
+                        {
+                            lang: "ukr",
+                            value: "Тест"
+                        }
+                        ]
+                        categoryCode: "23131"
+                        subcategories: [{
+                            categoryCode: "22341"
+                            name: [{
+                                lang: "eng",
+                                value: "Test"
+                            },
+                            {
+                                lang: "ukr",
+                                value: "Тест"
+                            }
+                            ]
+                            available: true
+                        }]
+                        available: true
+                    }){
+                        _id
+                        name {
+                            lang
+                            value
+                        }
+                        categoryCode
+                        subcategories {
+                            categoryCode
+                            name {
+                                lang
+                                value
+                            }
+                            available
+                        }
+                        available
+                    }
+                }
+            `
+        })
+
+        categoryId = res.data.addCategory._id
+        
+        expect(categoryService.addCategory({
```

Comment:
Please, avoid using direct service calls as much as possible. The idea of this kind of test is that you make HTTP call and check the response that is returned from the backend. But we should not care about inner workings of the services and their methods. Tests should know nothing about it. Of course, there are some exceptions, for example, when we need to delete entity to clear data that is used in tests but operation DELETE is not provided by API.